contrib/google.golang.org/grpc.v12/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/signalfx/signalfx-go-tracing/contrib/google.golang.org/grpc.v12
```

Comment:
This is only to be copied from `.circleci` during test runs, correct?

Reply:
Oh yes. Need to remove this.